numpyro/contrib/module.py

```diff
@@ -49,12 +49,16 @@ def flax_module(name, nn_module, *, input_shape=None, **kwargs):
         args = (jnp.ones(input_shape),) if input_shape is not None else ()
         # feed in dummy data to init params
         rng_key = numpyro.prng_key()
-        _, nn_params = nn_module.init(rng_key, *args, **kwargs)
+        nn_params = nn_module.init(rng_key, *args, **kwargs)
+
+        # make a mutable copy
+        nn_params = flax.core.unfreeze(nn_params)
```

Comment:
You can capture parameters with `nn_params = flax.core.unfreeze(nn_params)["params"]`